scripts/down.sh

```diff
@@ -3,4 +3,4 @@
 set -e
 
 docker-compose down
-sudo rm -rf db_backup db_data
+rm -rf db_backup db_data
```

Comment:
I guess this is not needed anymore. If it is the case, we can safely remove the script and directly use `docker-compose down` instead of calling it.

Reply:
we can keep the script, but yes it only does a `docker-compose down -v` (with the -v to delete the volumes)